src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -430,6 +420,18 @@ public override Task<string> GetCacheFilePathAsync()
             return Task.FromResult(NoOpRestoreUtilities.GetProjectCacheFilePath(cacheRoot: spec.RestoreMetadata.OutputPath));
         }
 
+        internal override bool IsCacheHit(bool cacheHitTargets, bool cacheHitPackageSpec, PackageSpec actual, PackageSpec last, FileInfo fileInfo)
```

Comment:
The method name does not seem to match what the method returns. 

This is really an up to date check.

Reply:
Fixed.